index.html

```diff
@@ -84,6 +84,29 @@ <h1>My Fall 2025 course list</h1>
                     I will need <strong style="color: red">15 credits</strong> in the Winter 2025 semester to graduate on time. All my core &amp; concentration courses will be done, so I will have only free option courses left to complete<br>
                 </p>
 
+                <div class="slideshow">
+
+                    <div class="slideshow_img">
+                        <img src="pictures/acropole.jpg" alt="Acropolis">
+                    </div>
+
+                    <div class="slideshow_img">
+                        <img src="pictures/lisbon.jpg" alt="Lisbon">
+                    </div>
+
+                    <div class="slideshow_img">
+                        <img src="pictures/milan.jpg" alt="Milan">
+                    </div>
+
+                    <div class="slideshow_img">
+                        <img src="pictures/hydra.jpg" alt="Hydra Greece">
+                    </div>
+
+                    <a id="prev" onclick="previous()">&lt;&lt;&lt;</a>
+                    <a id="next" onclick="next()">&gt;&gt;&gt;</a>
+                </div> 
```

Comment:
Works well! But it's a bit big on my screen, I can't see the whole picture. I would have resized a little, maybe.